### PR TITLE
Changed the build file to reflect the move to suffuse

### PR DESCRIPTION
**Pull request by EECOLOR**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.kateproject*
```
Comment (paulp) [inline .gitignore]: .kateproject*?

Comment (EECOLOR) [inline .gitignore]: I'll remove it from this commit.

Comment (EECOLOR) [inline .gitignore]: I totall forgot about the global git ignore

**Comment by paulp** [comment]
I fixed the conflict and merged it.

**Pull request closed without merging**